web.py

```diff
@@ -213,6 +214,32 @@ def test_ws():
             flash('Event not yet implemented', 'danger')
     return render_template('test_ws.html', recipient=recipient, event=event, events=events)
 
+@roles_accepted(Role.ROLE_ADMIN, Role.ROLE_FINANCE)
+@app.route('/user_kyc', methods=['GET', 'POST'])
+def user_kyc():
+    email = ''
+    if request.method == 'POST':
+        email = request.form['email']
+        user = User.from_email(db.session, email)
+        if user:
+            kycrequest = KycRequest.from_user(db.session, user)
+            if kycrequest.status == 'completed':
```

Comment:
dont use a string literal use the constant `STATUS_COMPETED`

Reply:
changed as per above